charts/kong-collectorapi/requirements.yaml

```diff
@@ -1,9 +1,9 @@
 dependencies:
 - name: postgresql
   version: 8.2.1
```

Comment:
Sync this version up from Kong's chart. Keeping them the same helps avoid any discrepancies.